code/modules/clothing/spacesuits/rig/suits/merc.dm

```diff
@@ -115,4 +115,25 @@
 		/obj/item/rig_module/fabricator/energy_net,
 		/obj/item/rig_module/device/door_hack,
 		/obj/item/rig_module/actuators/combat
-		)
\ No newline at end of file
+		)
+
+/obj/item/rig/merc/einstein //For the Einstein Merc kit
+	name = "paragon suit control module"
+	desc = "A back mounted control mechanism of an Einstein Engines hardsuit. This model is issued to the leaders of security teams within the corporation."
+	suit_type = "paragon"
+	icon = 'icons/clothing/rig/einstein_merc.dmi'
+	icon_state = "paragon"
+	icon_supported_species_tags = null
+	allowed_module_types = MODULE_GENERAL | MODULE_LIGHT_COMBAT | MODULE_HEAVY_COMBAT | MODULE_SPECIAL | MODULE_MEDICAL | MODULE_UTILITY
+	species_restricted = list(BODYTYPE_HUMAN, BODYTYPE_IPC, BODYTYPE_IPC_BISHOP, BODYTYPE_IPC_ZENGHU)
+
+	initial_modules = list(
+		/obj/item/rig_module/power_sink,
+		/obj/item/rig_module/mounted/ion,
```

Comment:
I am a bit unsure about the mounted ion rifle, as ion weapons are not easily available during standard gameplay and emps arnt exactly balanced atm.